products/sle/main.pm

```diff
@@ -839,8 +839,8 @@ sub load_hacluster_tests() {
         loadtest("ha/crm_mon.pm");
         loadtest("ha/fencing.pm");
         if (!get_var("HACLUSTERJOIN")) {                     #node1 will be fenced
-            loadtest "ha/fencing_boot.pm";
-            loadtest "ha/fencing_consoletest_setup.pm";
+            loadtest "installation/first_boot.pm";
+            loadtest "console/consoletest_setup.pm";
```

Comment:
not a good idea as this will overwrite modules in openQA details and look messed up. We can not yet properly load the same module multiple times in the same test plan. Either we need to fix openQA (os-autoinst already renames the module on second load) or use this symlink workaround